### PR TITLE
Add repo

### PR DESCRIPTION
**Pull request by kyle-long**
I called it `add-repo` but I am thinking maybe I should have called it `add-repository`.  Any thought on that?

### CODE REVIEW

File: formulas/add-repo/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Add Repo
+========
+Exposes functions for adding a repository to a package management system.
```
Comment (fidian) [inline formulas/add-repo/README.md]: Newline above

File: formulas/add-repo/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+    wickFormula add-repo
+
+Returns nothing.
+
+
+`addRepoApt()`
+--------------
+
+External: Adds a repository to apt.  If we find that you are running an operating system that should not be using apt we do not attempt to add it.
```
Comment (fidian) [inline formulas/add-repo/README.md]: Drop the "External:" bit.  It's implied.

Comment (kyle-long) [inline formulas/add-repo/README.md]: oh.  I did not know that.  I have been adding them on other functions.  I will stop doing that.

9ba4a62a111dd03a002badde6ddca8ca77199069

File: formulas/add-repo/README.md (hunk 1 above)
Comment (kyle-long) [inline formulas/add-repo/README.md]: 6aa5df3506c43439ae8e1d1b2514eeaea268df87

**Comment by kyle-long** [comment]
Rename

c7c3a1d2296428dd673c11d93d44ca78843b3940
a22beb672b7a4cea385f9416fcacda22d397be53
8e8826d88e0e05b77415ca9d1d5d156234bbb0d1

**Comment by fidian** [comment]
Thanks for this commit!

**Pull request merged**